lib/Doctrine/DBAL/Driver/PDOConnection.php

```diff
@@ -48,17 +52,17 @@ public function exec($statement)
      */
     public function getServerVersion()
     {
-        return PDO::getAttribute(PDO::ATTR_SERVER_VERSION);
+        return $this->conn->getAttribute(PDO::ATTR_SERVER_VERSION);
     }
 
     /**
      * {@inheritdoc}
      */
-    public function prepare($prepareString, $driverOptions = [])
+    public function prepare($prepareString)
```

Comment:
No longer needed to follow the `PDO` signature. Will add to the changelog later.